src/main/java/duke/command/UnknownCommand.java

```diff
@@ -0,0 +1,14 @@
+package duke.command;
+
+import duke.main.Ui;
+import duke.utils.Storage;
+import duke.utils.TaskList;
+
+public class UnknownCommand implements Command {
```

Comment:
Perhaps this could be a subclass of Exception? However, I understand your viewpoint as to why you'd prefer a class for this matter. Again, it is down to personal preference.